pusher-http-haskell.cabal

```diff
@@ -41,7 +41,8 @@ library
                       text ==1.2.*,
                       time >=1.5 && <1.7,
                       transformers >=0.4 && <0.6,
-                      unordered-containers ==0.2.*
+                      unordered-containers ==0.2.*,
+                      vector
```

Comment:
Can you add version bounds to this. For now just make it =0.11.*. This can prevent breaking changes if a newer version of vector would be pulled in which this library does not support.